doc.html

```diff
@@ -77,16 +94,13 @@ <h2>5.1 Vineyard Irrigation</h2>
 
 Wine grape vines are a delicate crop, that can produce new vegetation and ripening grapes at the same time, with substantially different water needs during its’ various growing stages.
 
-An irrigation_advisor service can be designed to promote regulated deficit irrigation over full irrigation, in certain growing stages (e.g., veraison stage), aiming to control the vegetation growth and improve grapes quality characteristics (e.g., sugar, pH, etc.).
-
-The service receives initially a request to prepare an irrigation application map for a vineyard plantation. Based on the vines growing stage (linked to growing degree days after budburst), the soil's properties and previously applied irrigations, the irrigation_advisor service determines the optimal irrigation amount, for a given application day, that will maintain the vineyard plants under the proper water status (e.g., well irrigated or mildly stressed).
+ADIS may ask from an irrigation_advisor to promote regulated deficit irrigation over full irrigation, in certain growing stages (e.g., veraison stage), aiming to control the vegetation growth and improve grapes quality characteristics (e.g., sugar, pH, etc.).
 
-The irrigation map is delivered on an irrigation management application vendor that is responsible (a) to convert the irrigation plan data to a format understandable by the equipment that will perform the irrigation application;(b) send the implemented irrigation application map to the
+If the field monitoring functionality is supported, the irrigation_advisor service calculates the water depletion within the root zone in predefined time intervals (can vary from hourly to daily depending on the irrigation_advisor implementation), and prepare an irrigation recommendation(s) when a certain depletion criterion is met.
 
-<h2>5.2 Tractor Irrigation</h2>
+When the advisor receives a prepare_advice request it may take into account the vines growing stage (linked to growing degree days after budburst), the soil's properties and previously applied irrigations, to determine the optimal irrigation amount, that will maintain the vineyard plants under the proper water status (e.g., well irrigated or mildly stressed). An advice preparation takes always into account the latest available data and it's validity is time-limited (see 9.1 Advice Preparation).
```

Comment:
"it may take into account the vines growing stage (linked to growing degree days after budburst) ..."
=> too implementation specific, maybe just "it may take into account the crop growing stage ..."

"that will maintain the vineyard plants"
=> again, too specific, use "that will maintain the crop"